phoenix-core/src/main/java/org/apache/phoenix/schema/transform/Transform.java

```diff
@@ -520,13 +537,28 @@ public static void doCutover(PhoenixConnection connection, SystemTransformRecord
             int batchSize = 0;
             for (TableInfo view : childViewsResult.getLinks()) {
                 String changeView = String.format(changeViewStmt,
-                        (columnNames.size() > 0? "," + String.join(",", columnNames):""),
-                        (view.getTenantId()==null || view.getTenantId().length == 0? null: ("'" + Bytes.toString(view.getTenantId()) + "'")),
-                        (view.getSchemaName()==null || view.getSchemaName().length == 0? null : ("'" + Bytes.toString(view.getSchemaName()) + "'")),
-                        Bytes.toString(view.getTableName()),
-                        (columnValues.size() > 0? "," + String.join(",", columnValues):""));
+                    columnNames.size() > 0 ? "," + String.join(",", columnNames) : "",
```

Comment:
Same list handling as before.
(I think we may already have some helper functions for that)